sockeye/lr_scheduler.py

```diff
@@ -13,7 +13,10 @@
 
 import logging
 from math import sqrt
-from typing import Optional
+from typing import Any, Dict, List, Optional, Tuple, Type
+
+import torch
```

Comment:
Elsewhere we import as `pt`, while non-standard, we could maybe apply the same import of `pt` here.

Reply:
This is an open question. Right now some parts of our code use `torch` and others use `pt`. Do we have a compelling reason to use `pt` instead of the standard `torch`?

Reply:
When we wrote the initial PyTorch version we preferred `pt` over `torch` because its shorter, requiring less line breaks, and follows the 'convention' of abbreviating math/ml libraries with 2 letters (mxnet -> mx, numpy -> np). I'd be in favor of using `pt`.

Reply:
I'm generally in favor of using libraries' chosen names. In their own documentation, [NumPy prefers `np`](https://numpy.org/doc/stable/user/quickstart.html) while [PyTorch prefers `torch`](https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html). This also avoids the need to change all instances of `torch` to `pt` during development (for example, when pasting snippets from both Sockeye and the PyTorch docs into the Python interpreter for interactive testing).

I agree with you about finding good ways to shorten lines. When we're using a line width of 120 characters, I'm not sure that +/- 3 makes a big difference.

Reply:
Depending on the statement, the difference might be more than 3 characters, but I see your point about pasting code snippets from PyTorch documentation.
If we want to make the change from `pt` to `torch`, we should do it consistently throughout the codebase. Maybe its worth doing this in one step in another PR?

Reply:
I don't have a strong preference either way.  I agree that we should do a pass to do this consistently in the codebase.

Reply:
I agree with changing everything over in a single, separate PR.

This PR uses the existing name in each source file (`pt` or `torch`). For the special case where the import needs to be added for `lr_scheduler.py`, it uses `torch` to match the training code that calls it.